MediaBrowser.Model/Entities/ProviderIdsExtensions.cs

```diff
@@ -45,11 +47,35 @@ public static bool HasProviderId(this IHasProviderIds instance, MetadataProvider
 
             if (instance.ProviderIds == null)
             {
-                return null;
+                id = string.Empty;
+                return false;
             }
 
-            instance.ProviderIds.TryGetValue(name, out string? id);
-            return string.IsNullOrEmpty(id) ? null : id;
+            return instance.ProviderIds.TryGetValue(name, out id);
```

Comment:
This change can break providers that are explicitly expecting null instead of an empty string

Reply:
I've done a quick search and couldn't find any.